Fix79 blocked by anchor

briankung opened the pull request:
Removes clickable cards. It's not worth the additional complexity/hacks right now, and the titles are actually quite clickable if they're not simply numbers.

Closes #79

briankung committed:
Removes coffeescript files

briankung committed:
Removes css

briankung committed:
Removes updated on date

briankung committed:
Changes class name, position of link

briankung committed:
Adds some jquery to handle clickable cards
This makes it so that you can click everything above the panel body.

briankung committed:
Adds .row to whitelisted clickable nodes

briankung committed:
Refactors paper tabs

briankung committed:
Uses proper tab controls

briankung committed:
Merge branch 'refactorPaperTabs' into fix79BlockedByAnchor

briankung committed:
Cleans up edit link_to

briankung commented:
Also some unrelated refactorings

wsf246 commented:
Unrelated Refactorings is my favorite band.

On Wed, Sep 14, 2016 at 5:35 PM, Brian Kung notifications@github.com
wrote:

> Also some unrelated refactorings
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> https://github.com/briankung/projectcredo/pull/80#issuecomment-247178089,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/ACKsZHGNRd5fE6hhpekzcBsgClMrgyBhks5qqHa2gaJpZM4J9PHE
> .

briankung committed:
Removes row and whitelist

briankung committed:
Removes clickable cards
It's not worth the additional complexity/hacks right now, and the titles are actually quite clickable if they're not simply numbers.

briankung committed:
Reverts more changes for clickable links

briankung committed:
Damn it, whitespace

briankung committed:
Okay, this is kind of anal.

briankung commented:
This has been merged into #81
